crates/engine/src/prepare/trusted_documents.rs

```diff
@@ -215,3 +357,47 @@ async fn handle_apq<'r, 'f>(
         ErrorCode::PersistedQueryNotFound,
     ))
 }
+
+/// When a request contains a trusted document id, but the trusted document is not found in GDN. Default: INFO.
+fn log_unknown_trusted_document_id<R: Runtime>(engine: &Engine<R>, document_id: &str) {
+    const MESSAGE: &str = "Unknown trusted document";
+
+    match engine.schema.settings.trusted_documents.document_id_unknown_log_level {
```

Comment:
you could have a `Into<tracing::Level>` and use `tracing::event!(level, "Unknown trusted document", document_Id)` instead

Reply:
I tried that first. The level has to be a compile-time constant, so that doesn't work.

Reply:
hum indeed, sad